elkscmd/misc_utils/Makefile

```diff
@@ -6,29 +6,7 @@ include $(BASEDIR)/Make.defs
 #
 # Include standard packaging commands.
 
-MIN_BIN 	= 
-MIN_LIB 	= 
-MIN_SBIN	= 
-MIN_USR_BIN	= 
-MIN_USR_LIB	= 
-MIN_USR_MAN	= 
-MIN_USR_SBIN	= 
-
-NET_BIN 	= 
-NET_LIB 	= 
-NET_SBIN	= 
-NET_USR_BIN	= 
-NET_USR_LIB	= 
-NET_USR_MAN	= 
-NET_USR_SBIN	= 
-
-STD_BIN 	= 
-STD_LIB 	= 
-STD_SBIN	= 
-STD_USR_BIN	= ed tar compress uncompress zcat miniterm
-STD_USR_LIB	= 
-STD_USR_MAN	= compress.1 uncompress.1 zcat.1
-STD_USR_SBIN	= 
+STD_USR_BIN    = ed tar compress uncompress zcat miniterm
```

Comment:
You can probably convert this to the PRGS variable used by most of the other applet directories.